docs/api-network.rst

```diff
@@ -2184,13 +2184,11 @@ Chain Methods
         >>> chain.time()
         1550189143
 
-.. py:method:: Chain.mine(blocks=1, timestamp=None, timedelta=None)
```

Comment:
`timestamp` and `timedelta` should still work when using ganache / hardhat so removing the documentation around it feels wrong.

Reply:
updated!